src/Hl7.Fhir.Serialization.Tests/TestData/json-edge-cases.json

```diff
@@ -1,201 +1,168 @@
 {
```

Comment:
Do you have an idea why this file has changed so much?  Is it just whitespace (or I seem some encoding of &nbsp; to 0x202f?

Reply:
It is mostly white space. For example:
it was:
```
"identifier": [
        {
          "system": "urn:ietf:rfc:3986",
          "value": "2.16.840.1.113883.19.5"
        }
      ],
```
and now it is:
```
"identifier": [{
                "system": "urn:ietf:rfc:3986",
                "value": "2.16.840.1.113883.19.5"
            }],
```

But during the download, something went wrong, I see now. The family name of the contact should be `du Marché`, but is now `du MarchÃ©`. So I think the encoding is not right in the download phase. I will have a look at it.